zerver/tests/test_message_edit.py

```diff
@@ -395,76 +395,69 @@ def test_user_info_for_updates(self) -> None:
     def test_edit_cases(self) -> None:
         """This test verifies the accuracy of construction of Zulip's edit
         history data structures."""
-        self.login('hamlet')
         hamlet = self.example_user('hamlet')
-        msg_id = self.send_stream_message(self.example_user("hamlet"), "Scotland",
-                                          topic_name="topic 1", content="content 1")
-        result = self.client_patch("/json/messages/" + str(msg_id), {
-            'message_id': msg_id,
-            'content': 'content 2',
-        })
-        self.assert_json_success(result)
-        history = orjson.loads(Message.objects.get(id=msg_id).edit_history)
-        self.assertEqual(history[0]['prev_content'], 'content 1')
-        self.assertEqual(history[0]['user_id'], hamlet.id)
-        self.assertEqual(set(history[0].keys()),
-                         {'timestamp', 'prev_content', 'user_id',
-                          'prev_rendered_content', 'prev_rendered_content_version'})
+        iago = self.example_user('iago')
```

Comment:
I'd explain in this commit the purpose/structure of the refactor / how to read it; it's not immediately obviously on skimming.

Reply:
Whoops, I meant to put a TODO in the body; definitely.